SimMuon/RPCDigitizer/python/muonRPCDigis_cfi.py

```diff
@@ -33,54 +33,55 @@
 
 #the digitizer for PhaseII muon upgrade is RPCSimModelTiming and for the moment is based on  RPCSimAverageNoiseEffCls
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
-if fastSim.isChosen():
-    simMuonRPCDigis.InputCollection = 'MuonSimHitsMuonRPCHits'
+fastSim.toModify(simMuonRPCDigis, InputCollection = 'MuonSimHitsMuonRPCHits')
+
+simMuonRPCDigisPhaseII = cms.EDProducer("RPCandIRPCDigiProducer",
```

Comment:
Better to start this variable name with an underscore, so there won't be an unnecessary `process.simMuonRPCDigisPhaseII` object.